packages/asset-service/src/service/TrustWalletService.ts

```diff
@@ -17,7 +17,11 @@ export const generateTrustWalletUrl = (assetId: AssetId) => {
     url += `/assets/`
     switch (chainNamespace) {
       case CHAIN_NAMESPACE.Evm:
-        url += Web3.utils.toChecksumAddress(assetReference)
+        // Skip checksum if we're hitting an EVM chain's native asset
+        // since native assets don't have an address
+        if (assetReference !== '60') {
```

Comment:
I think we should actually remove `ethereum` from `ethAssets`
https://github.com/shapeshift/web/blob/polygon-support/packages/asset-service/src/generateAssetData/ethereum/index.ts#L47

we have the icon hardcoded from `baseAssets` and don't need to try to get an icon from trust wallet or generate and identicon for it.

it is being returned in the asset list here: https://github.com/shapeshift/web/blob/polygon-support/packages/asset-service/src/generateAssetData/ethereum/index.ts#L96